pkg/agent/apiserver/handlers/podinterface/handler.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podinterface
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+// Response describes the internal response struct of the pod command.
+type Response struct {
+	PodName       string `json:"name,omitempty" yaml:"name,omitempty" antctl:"name,The name of the pod"`
+	InterfaceName string `json:"InterfaceName,omitempty" yaml:"InterfaceName,omitempty"`
+	IP            string `json:"IP,omitempty" yaml:"IP,omitempty"`
+	MAC           string `json:"MAC,omitempty" yaml:"MAC,omitempty"`
+	PortUUID      string `json:"PortUUID,omitempty" yaml:"PortUUID,omitempty"`
+	OFPort        int32  `json:"OFPort,omitempty" yaml:"OFPort,omitempty"`
+	ContainerID   string `json:"ContainerID,omitempty" yaml:"ContainerID,omitempty"`
+	PodNamespace  string `json:"PodNamespace,omitempty" yaml:"PodNamespace,omitempty"`
```

Comment:
All other tags in antrea and K8s are not capitalized, is there any reason the fields except name are capitalized?

Reply:
You are right. Code updated.